docs/user/geojson.rst

```diff
@@ -0,0 +1,223 @@
+JSON and GeoJSON
+================
+
+GeoMesa provides built-in integration with JSON and GeoJSON. JSON can be stored as an attribute in a simple feature
+and queried using JSONPath through CQL. For users who are not interested in GeoTools, GeoMesa provides a GeoJSON
+API that allows for ingesting and querying with pure JSON.
+
+.. _index_structure:
+
+JSON Attributes
+---------------
+
+GeoMesa allows simple feature attributes to be specified as JSON. JSON attributes must be specified when
```

Comment:
This first sentence is somewhat ambiguously worded. You mean to say that simple feature attributes can be specified to have a "type" of JSON (or annotated that they contain JSON), but this makes it sound like you can define the  attribute itself *using* JSON. (Well, I suppose you can with the convert library, but that's not the point of this feature.)